proof/crefine/AARCH64/IpcCancel_C.thy

```diff
@@ -990,13 +626,248 @@ lemma c_invert_assist: "3 - (ucast (p :: priority) >> 6 :: machine_word) < 4"
   using prio_ucast_shiftr_wordRadix_helper'[simplified wordRadix_def]
   by - (rule word_less_imp_diff_less, simp_all)
 
+lemma addToBitmap_ccorres:
```

Comment:
out of curiosity, did you copy these from elsewhere, or redo?

Reply:
`addToBitmap_corres` and `removeFromBitmap_corres` are new. I did the first and you did the second. Previously they were all jumbled up in the proofs of the `corres` proofs for the ready queue functions

Reply:
Ah, that's why it fell into the uncanny valley of memory. Cool.